src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -187,6 +193,40 @@ export /* istanbul ignore next */ function PipelineRuns(props) {
         }),
         action: rerun
       },
+      {
+        actionText: intl.formatMessage({
+          id: 'dashboard.startPipelineRun.actionText',
+          defaultMessage: 'Start'
+        }),
+        action: start,
+        disable: resource => {
+          const { reason, status } = getStatus(resource);
+          return !isPending(reason, status);
+        },
+        modalProperties: {
```

Comment:
I think we can leave out the `modalProperties` here, no need to confirm with the user since they're not deleting anything or performing an action they can't cancel or otherwise recover from.